### PR TITLE
Update borgbackup-1.2.4.tar.gz to 1.4.0

### PR DESCRIPTION
**Pull request by shivansh02**

### CODE REVIEW

**Comment by shivansh02** [comment]
@Hofer-Julian Please take a look!

**Comment by Hofer-Julian** [comment]
Hey @shivansh02, as far as I can tell https://github.com/borgbase/vorta/issues/1794 is not yet resolved. As far as I can tell, without a migration procedure some user would see their backups failing after we would upgrade borg. Or is this handled nowadays?

**Comment by m3nu** [comment]
We won't implement https://github.com/borgbase/vorta/issues/1794, since too many things can go wrong to automate it within Vorta. And it seems to affect very few repos. I've seen virtually no questions or reports from users or my own repos.

File: dependencies/borgbackup.json [hunk 1]
```diff
@@ -7,13 +7,13 @@
...
-            "sha256": "a4bd54e9469e81b7a30a6711423115abc818d9cd844ecb1ca0e6104bc5374da8",
+            "url": "https://files.pythonhosted.org/packages/dd/0d/28e60180ce4ae171adba65ce9f8878fce3580c6d2cfdfa998929175105dd/borgbackup-1.4.0.tar.gz",
+            "sha256": "c54c45155643fa66fed7f9ff2d134ea0a58d0ac197c18781ddc2fb236bf6ed29",
             "x-checker-data": {
                 "type": "pypi",
                 "name": "borgbackup",
                 "versions": {
-                    "<": "1.2.5"
+                    "<": "1.4.1"
                 }
```
Comment (Hofer-Julian) [inline dependencies/borgbackup.json:17]: This is not needed anymore

**Review by Hofer-Julian — CHANGES_REQUESTED** [review]

**Comment by Hofer-Julian** [comment]
> I've seen virtually no questions or reports from users or my own repos.

If that's the case then I think there's no need to block here.

**Review by Hofer-Julian — APPROVED** [review]

**Comment by Hofer-Julian** [comment]
Thanks @shivansh02!

**Comment by m3nu** [comment]
Thank to the both of you!

**Pull request merged**